lib/ansible/module_utils/netapp.py

```diff
@@ -226,7 +229,7 @@ def request(url, data=None, headers=None, method='GET', use_proxy=True,
         return resp_code, data
 
 
-def ems_log_event(source, server, name="Ansible", id="12345", version="1.1",
+def ems_log_event(source, server, name="Ansible", id="12345", version="2.7",
```

Comment:
Is this is meant to represent the Ansible version it should be read from `lib/ansible/release.py`, rather than hardcoding the version in this file

Reply:
Talked with the team we are going to use __version__ from release.py, the change is currently in out internal CI pipeline and waiting on code review, once that done i'll push it out to github.